packages/reqresp/test/unit/encodingStrategies/sszSnappy/snappyFrames/uncompress.test.ts

```diff
@@ -56,4 +57,43 @@ describe("encodingStrategies / sszSnappy / snappy frames / uncompress", () => {
 
     expect(decompress.uncompress(new Uint8ArrayList(Buffer.alloc(3, 1)))).toBe(null);
   });
+
+  it("should detect invalid checksum", () => {
+    const chunks = new Uint8ArrayList();
+    chunks.append(IDENTIFIER_FRAME);
+
+    chunks.append(Uint8Array.from([ChunkType.UNCOMPRESSED, 0x80, 0x00, 0x00]));
+    // first 4 bytes are checksum
+    // 0xffffffff is clearly an invalid checksum
+    chunks.append(Uint8Array.from(Array.from({length: 0x80}, () => 0xff)));
```

Comment:
why not use 128 here?  Guessing this is a snappy spec thing but curious none-the-less

```suggestion
    chunks.append(Uint8Array.from(Array.from({length: 128}, () => 0xff)));
```

Reply:
just an artifact from a test case I had been provided, there's nothing special about 128.